docs/cli/newrelic_entity_changetracker_create.md

```diff
@@ -0,0 +1,52 @@
+## newrelic entity changetracker create
+
+Create a change to the given entity
+
+### Synopsis
+
+Create a change to the given entity
+
+The create command marks a change to the given New Relic entity.
+
+
+```
+newrelic entity changetracker create [flags]
+```
+
+### Examples
+
+```
+newrelic entity changetracker create --guid <GUID> --version <1.0.0>
+```
+
+### Options
+
+```
+  -h, --help                    help for create
+  -g, --guid string             the entity GUID to create change tracker
+  -v, --version string          the tag names to add to the entity
```

Comment:
For `version` and the other fields, perhaps we should copy as close as practical [the docs that we have for the graphql api](https://source.datanerd.us/markers/deployment-marker-api/blob/2da074e90d835b3d01f4be021294e95d4aa57fe9/src/main/resources/schema.graphql#L29-L50)?

> The version of the deployed software, for example, something like v1.1

Reply:
Also, can we indicate that `version` is required? And include the necessary validation that it's a non-blank value?